exercism/ocr-numbers/ocr-numbers.js

```diff
@@ -0,0 +1,27 @@
+var convert = function(stringNum) {
+  var lookup = {
+    ' _ \n| |\n|_|\n   ': '0',
+    '   \n  |\n  |\n   ': '1',
+    ' _ \n _|\n|_ \n   ': '2',
+    ' _ \n _|\n _|\n   ': '3',
+    '   \n|_|\n  |\n   ': '4',
+    ' _ \n|_ \n _|\n   ': '5',
+    ' _ \n|_ \n|_|\n   ': '6',
+    ' _ \n  |\n  |\n   ': '7',
+    ' _ \n|_|\n|_|\n   ': '8',
+    ' _ \n|_|\n _|\n   ': '9',
+    '    _ \n  || |\n  ||_|\n      ': '10',
+    '   \n| |\n| |\n   ': '?',
```

Comment:
The spec said that garbled numbers were supposed to be replaced with `'?'`.  This code changes a specific garbled number, not all garbled numbers, so it doesn't meet the spec.